metadata-models/src/main/pegasus/com/linkedin/ml/metadata/ModelProperties.pdl

```diff
@@ -0,0 +1,39 @@
+namespace com.linkedin.ml.metadata
+
+import com.linkedin.common.FeatureUrn
+
+/**
+ * Properties associated with a ML Model
+ */
+record ModelProperties {
+
+  /**
+   * Documentation of the ML Model
+   */
+  description: optional string
+
+  /**
+   * Date when the model was developed
+   */
+  date: optional string
```

Comment:
If you want to capture both time and the creator then `com.linkedin.common.AuditStamp` might be better?

Reply:
Chose to use `com.linkedin.common.Time` model here per @hshahoss's comment.